src/coreComponents/fileIO/silo/SiloFile.cpp

```diff
@@ -1335,6 +1335,8 @@ static std::vector< int > getSiloNodeOrdering( ElementType const elementType )
     case ElementType::Pyramid:       return { 0, 2, 3, 1, 4 };
     case ElementType::Wedge:         return { 1, 0, 2, 3, 5, 4 };
     case ElementType::Hexahedron:    return { 0, 1, 3, 2, 4, 5, 7, 6 };
+    case ElementType::Prism5:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism6:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
```

Comment:
Seeing the TODO, I am just asking out of curiosity if prisms are supported in Silo? But no need to change anything in the code

Reply:
Silo does not have a native type for pentagonal and hexagonal prisms. They need to be output as general polyhedra (similarly to VTK). This will be addressed in a follow-up PR introducing general prismatic elements.